### ISSUE TITLE
Clarify what the scales are

### ISSUE THREAD

**Issue by steadicat** [boronine/colorspaces.js#9]
It's not exactly clear to me from the docs or the site what the ranges are for the various variables in each of the color spaces. For example, I assumed that CIELCHuv would be 0-360 for H, and 0-100 for L and C.

Indeed it seems like maximum lightness (i.e. the lightness of white), is 100:

```JS
> var rgb = colorspaces.converter('sRGB', 'CIELCHuv');
> rgb([1, 1, 1])
[ 100, 0.017131122100079495, 272.9802266955604 ]
```

This could be clearer in the docs.

But then  I noticed the following:

```JS
> var rgb = colorspaces.converter('sRGB', 'CIELCHuv');
> rgb([1, 0, 0])
[ 53.23288178584245, 179.077262517562, 12.16957162567793 ]
> rgb([0, 1, 0])
[ 87.73703347354422, 135.78401352585232, 127.72355232980077 ]
> rgb([0, 0, 1])
[ 32.302586667249486, 130.69687033510064, 265.87566146568497 ]
```

For pure sRGB green and blue, the chroma exceeds 130, and for sRGB red the chroma is close to 180! Is there such a thing as “maximum chroma”? If so, what is it?

If there is no absolute max chroma, perhaps there could be a way of computing what the maximum value is given the other two parameters while remaining within sRGB? For example, there could be a call like this:

```JS
> var lchToRgb = colorspaces.converter('CIELCHuv', 'sRGB');
> lchToRgb.maxC(/* L */ 53, /* H */ 12)]);
177.70552
```

**Comment by boronine** [comment]
Hi Stefano, sorry for the delay.

> This could be clearer in the docs.

Agreed. I expanded the Color Spaces section in the README to address this.

> Is there such a thing as “maximum chroma”?

There isn't!

> If there is no absolute max chroma, perhaps there could be a way of computing what the maximum value is given the other two parameters while remaining within sRGB?

It's funny to read your code snippet, I actually implement that *exact* function in a separate project: https://github.com/husl-colors/husl/blob/17e12bfc1aa867c295b4f3d9470a6e0ee2a75af3/husl.js#L122

I created [HUSL](http://www.husl-colors.org/) when I faced the exact problem you described, undefined chroma range. What I wanted was to generate colors in a perceptually uniform color space (CIELUV) and convert them to RGB for display. 

The function `maxChromaForLH` is exported, but I would recommend looking at HUSL's public API instead.
